Snowball-Finance/axial-frontend
# Ui-integration of staking page wrt design

To test:
1. Test the design for staking page as per figma https://www.figma.com/file/aYVD9iZnfmsEKzlN2I3dtg/Axial-V2-UI-Designs?node-id=3288%3A621734
2. Check the staking list item showing correct info(sAxial and veAxial)
3. Check the deposit flow on the sAxial and veAxial
4. Check the withdraw flow on the sAxial and veAxial

Need to work:
Responsiveness for mobile
